controllers/core/node_controller.go

```diff
@@ -66,6 +72,18 @@ func (r *NodeReconciler) Reconcile(ctx context.Context, req ctrl.Request) (ctrl.
 		return ctrl.Result{Requeue: true}, goErr.New("pod datastore hasn't been synced, node controller need wait to retry")
 	}
 
+	// CNINodes are created in default namespace
```

Comment:
Woah! I just realized this. Because its 1:1 with Node, which is a cluster scoped object, we should make sure these CRDs are cluster scoped.

Reply:
changed to cluster scope.